src/main/scala/com/databricks/spark/sql/perf/mllib/feature/QuantileDiscretizer.scala

```diff
@@ -0,0 +1,34 @@
+package com.databricks.spark.sql.perf.mllib.feature
+
+import org.apache.spark.ml
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.PipelineStage
+import org.apache.spark.sql._
+
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+import com.databricks.spark.sql.perf.mllib.{BenchmarkAlgorithm, MLBenchContext, TestFromTraining}
+
+/** Object for testing QuantileDiscretizer performance */
+object QuantileDiscretizer extends BenchmarkAlgorithm with TestFromTraining with UnaryTransformer {
```

Comment:
Would you mind adding numBuckets and relativeError as parameters?  numBuckets is especially important since the default value is tiny (2).